src/main/java/mate/project/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,24 @@
+package mate.project.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Objects;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String first;
```

Comment:
The 'first' and 'second' instance variables could have more descriptive names to improve code readability.